src/components/PopupWithForm.js

```diff
@@ -30,4 +32,12 @@ export default class PopupWithForm extends Popup {
       this.close();
     });
   }
+
+  setLoading(isLoading) {
```

Comment:
Great that you've implemented the logic for handling the loading via the class methods